pkg/controller/plan/kubevirt.go

```diff
@@ -1687,13 +1687,13 @@ func (r *KubeVirt) vmAllButMigrationLabels(vmRef ref.Ref) (labels map[string]str
 }
 
 // Represents a CDI DataVolume and add behavior.
-type DataVolume struct {
+type DataVolumeWrapper struct {
```

Comment:
it would be nice to drop it as it only holds the DV and PVC. or use it much more. for now the renaming is less confusing.

Reply:
renamed